solidity/test/token_stake/TestTokenStake.js

```diff
@@ -651,13 +677,12 @@ contract('TokenStaking', function(accounts) {
     })
   
     it("should report no eligible stake before initialization period is over", async () => {
-      await delegate(operatorOne, stakingAmount)
+      let tx = await delegate(operatorOne, stakingAmount)
+      let createdAt = (await web3.eth.getBlock(tx.receipt.blockNumber)).timestamp
       await stakingContract.authorizeOperatorContract(
         operatorOne, operatorContract, {from: authorizer}
       )
-  
-      await increaseTime(initializationPeriod);
-  
+      await increaseTimeTo(createdAt + initializationPeriod - 1)
```

Comment:
Why `-1`? I think this one should be `await increaseTimeTo(createdAt + initializationPeriod)`